From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paolo Sabatino <paolo.sabatino@gmail.com>
Date: Wed, 8 Mar 2023 11:12:41 +0100
Subject: [ARCHEOLOGY] rockchip64: set poll rate 50ms for rk3399 dmc driver

> X-Git-Archeology: - Revision 03a00c5ce49dadd0dd579980114b8595f144b763: https://github.com/armbian/build/commit/03a00c5ce49dadd0dd579980114b8595f144b763
> X-Git-Archeology:   Date: Wed, 08 Mar 2023 11:12:41 +0100
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip64: set poll rate 50ms for rk3399 dmc driver
> X-Git-Archeology:
> X-Git-Archeology: - Revision 34ae84fac5d0b66a1ab2d1e51534b7beb13ef245: https://github.com/armbian/build/commit/34ae84fac5d0b66a1ab2d1e51534b7beb13ef245
> X-Git-Archeology:   Date: Fri, 05 May 2023 14:22:00 +0200
> X-Git-Archeology:   From: amazingfate <liujianfeng1994@gmail.com>
> X-Git-Archeology:   Subject: bump rockchip64 edge to v6.3
> X-Git-Archeology:
---
 drivers/devfreq/rk3399_dmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/devfreq/rk3399_dmc.c b/drivers/devfreq/rk3399_dmc.c
index daff40702615..62f4804134c2 100644
--- a/drivers/devfreq/rk3399_dmc.c
+++ b/drivers/devfreq/rk3399_dmc.c
@@ -430,7 +430,7 @@ static int rk3399_dmcfreq_probe(struct platform_device *pdev)
 	dev_pm_opp_put(opp);
 
 	data->profile = (struct devfreq_dev_profile) {
-		.polling_ms	= 200,
+		.polling_ms	= 50,
 		.target		= rk3399_dmcfreq_target,
 		.get_dev_status	= rk3399_dmcfreq_get_dev_status,
 		.get_cur_freq	= rk3399_dmcfreq_get_cur_freq,
-- 
Armbian

